cmd/gateway-main.go

```diff
@@ -38,6 +38,8 @@ var (
 		Flags:           append(serverFlags, globalFlags...),
 		HideHelpCommand: true,
 	}
+
+	errLoginCredentailsNotSet = errors.New("Access and secret keys not set")
```

Comment:
We are in no need for a new error type for this situation since we are going to just error out anyways, so we can simply keep  error definition close to `errorIf()`